docs/azure_arc_jumpstart/azure_edge_iot_ops/aio_manufacturing/_index.md

```diff
@@ -0,0 +1,437 @@
+---
+type: docs
+title: "Azure Edge IoT Operations - Manufacturing"
+linkTitle: "Azure Edge IoT Operations - Manufacturing"
+weight: 3
+description: >
+---
+
+## Enhance operational insights at the edge using Azure Edge IoT Operations (AIO)
+
+Contoso Bakeries, a fictional company, requires dashboards and analytics on the Edge (in its plant) and in Azure to provide real-time operational data to plant operators such as Operational Technology (OT) Managers, Operators, and Maintenance personnel. This includes insights into production line operations and key performance indicators (KPIs) like overall equipment effectiveness (OEE), downtime, and waste. A prime focus is on data from critical equipments like fryers. Additionally, regional executives and directors are interested in supervising main KPIs across multiple plants remotely.
+
+## Architecture
+
+The solution for Contoso will involve deploying the following:
+
+- A Kubernetes cluster on the Edge using AKS Edge Essentials, managed by Azure Arc, which hosts various Azure IoT Operations services.
+- A custom-developed MQTT-Simulator that mimics plant and fryer data, transmitting it to Azure IoT MQ via MQTT.
+- An MQTT-listener service that captures this production line data, sending it to an open-source InfluxDb on the Edge, generating near real-time dashboards and analytics and visualizes it using Grafana.
+- An Azure IoT MQ connection with Azure Event Grid namespace, an MQTT broker integrated into Azure, pushing data to Azure Event Grid and subsequently to Azure Data Explorer.
+- Azure Data Explorer dashboards and analytics for supervisory and executive roles monitoring multiple plants' operations.
```

Comment:
```suggestion
The solution for Contoso will involve deploying the following:
- A Kubernetes cluster on the edge using AKS Edge Essentials, managed by Azure Arc, which hosts various Azure IoT Operations (AIO) services.
- A custom-developed MQTT-Simulator that mimics plant and fryer data, transmitting it to Azure IoT MQ via MQTT.
- An MQTT-listener service that captures this production line data, sending it to an open-source InfluxDb on the Edge, generating near real-time dashboards and analytics, and visualizing it using Grafana.
- An Azure IoT MQ connection with Azure Event Grid namespace, an MQTT broker integrated into Azure, pushing data to Azure Event Grid and subsequently to Azure Data Explorer.
- Azure Data Explorer dashboards and analytics for supervisory and executive roles monitoring multiple plants' operations.
```

Reply:
- No need to capitalize Edge in a sentence
- Missing link to AIO product docs, InfluxDB, Azure Data Explorer (check more)
- open source - no hyphen